packages/landing/next.config.js

```diff
@@ -0,0 +1,23 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  output: "export",
+  distDir: "../../dist",
```

Comment:
`output` can be the default one, as it will be a SSR app, `distDir` must be inside the package, different from the App `distDir`